redirects.yaml

```
@@ -315,3 +315,117 @@ redirects:
+  - source: /aria-required-children/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-required-children
+  - source: /aria-required-parent/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-required-parent
+  - source: /aria-roles/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-roles
+  - source: /aria-valid-attr-value/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr-value
+  - source: /aria-valid-attr/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr
+  - source: /duplicate-id-aria/
+    destination: https://dequeuniversity.com/rules/axe/4.4/duplicate-id-aria
+  - source: /button-name/
+    destination: https://dequeuniversity.com/rules/axe/4.4/aria-command-name
+  - source: /document-title/
+    destination: https://dequeuniversity.com/rules/axe/4.4/document-title
+  - source: /form-field-multiple-labels/
+    destination: https://dequeuniversity.com/rules/axe/4.4/form-field-multiple-labels
+  - source: /frame-title/
+    destination: https://dequeuniversity.com/rules/axe/4.4/frame-title
+  - source: /image-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/image-alt
+  - source: /input-image-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/input-image-alt
+  - source: /label/
+    destination: https://dequeuniversity.com/rules/axe/4.4/label
+  - source: /link-name/
+    destination: https://dequeuniversity.com/rules/axe/4.4/link-name
+  - source: /object-alt/
+    destination: https://dequeuniversity.com/rules/axe/4.4/object-alt
+  - source: /color-contrast/
+    destination: https://dequeuniversity.com/rules/axe/4.4/color-contrast
+  - source: /definition-list/
+    destination: https://dequeuniversity.com/rules/axe/4.4/definition-list
+  - source: /dlitem/
+    destination: https://dequeuniversity.com/rules/axe/4.4/dlitem
+  - source: /list/
+    destination: https://dequeuniversity.com/rules/axe/4.4/list
+  - source: /listitem/
+    destination: https://dequeuniversity.com/rules/axe/4.4/listitem
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /layout-table/
+    destination:
+  - source: /td-headers-attr/
+    destination: https://dequeuniversity.com/rules/axe/4.4/td-headers-attr
+  - source: /th-has-data-cells/
+    destination: https://dequeuniversity.com/rules/axe/4.4/th-has-data-cells
+  - source: /meta-refresh/
+    destination: https://dequeuniversity.com/rules/axe/4.4/meta-refresh
+  - source: /meta-viewport/
+    destination: https://dequeuniversity.com/rules/axe/4.4/meta-viewport
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /audio-caption/
+    destination:
+  - source: /video-caption/
+    destination: https://dequeuniversity.com/rules/axe/4.4/video-caption
+  # TODO - there doesn't seem to be a corresponding axe rule.
+  - source: /video-description/
+    destination:
```

```suggestion
  - source: /video-description/
    destination: https://dequeuniversity.com/rules/axe/3.3/video-description
```
removed in https://github.com/dequelabs/axe-core/pull/1737, axe-core 3.4